jquery-src/jquery.calendars.chinese.js

```diff
@@ -273,7 +273,7 @@
             var monthDaysTable = LUNAR_MONTH_DAYS[year - LUNAR_MONTH_DAYS[0]];
             var intercalaryMonth = monthDaysTable >> 13;
 
-            return (intercalaryMonth && intercalaryMonth === monthIndex);
+            return (intercalaryMonth !== 0 && intercalaryMonth === monthIndex);
```

Comment:
@alexcjohnson Actually, the change is a bit more intrusive.

We would go from:

```js
var isIntercalary = chineseCalendar.intercalaryMonth(year, monthIndex);
```

to:

```js
var isIntercalary = (monthIndex === chineseCalendar.intercalaryMonth(year));
```

I still think the change is for better.